akka-typed-tests/src/test/scala/akka/typed/cluster/ClusterApiSpec.scala

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.typed.cluster
+
+import akka.cluster.ClusterEvent._
+import akka.typed.TypedSpec
+import akka.typed.internal.adapter.ActorSystemAdapter
+import akka.typed.scaladsl.adapter._
+import akka.typed.testkit.TestKitSettings
+import akka.typed.testkit.scaladsl.TestProbe
+import com.typesafe.config.ConfigFactory
+import org.scalatest.concurrent.ScalaFutures
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+object ClusterApiSpec {
+  val config = ConfigFactory.parseString(
+    """
+      akka.actor.provider = cluster
+      akka.remote.artery.enabled = true
+      akka.remote.artery.canonical.port = 25552
```

Comment:
should be alright to use 0 here?

I think we should write these tests to work with both artery and classic, if possible, i.e.:
```
    akka.remote.netty.tcp.port = 0
    akka.remote.artery.canonical.port = 0
```